package.json

```diff
@@ -38,7 +39,9 @@
     "react-redux": "^5.0.6",
     "react-responsive": "^4.0.3",
     "react-router-dom": "^4.2.2",
-    "redux": "^3.7.2"
+    "redux": "^3.7.2",
+    "redux-logger": "^3.0.6",
+    "redux-saga": "^0.16.0"
```

Comment:
🎉  for +saga! 

fyi: there have also been some updates to some npm packages in package.json in PR 72, recently merged to master. but when you rebase master into `zubairnahmed:setup-redux-saga`, i am pretty confident you will not have any merge conflicts in `package.json` , because none of the things you've changed were changed in https://github.com/LearnersGuild/talent/pull/72/ . let me know if this is not the case.